src/components/gauge/gauge.scss

```diff
@@ -0,0 +1,222 @@
+@import "../common/_base";
+
+gx-gauge {
```

Comment:
I see there are multiple different values for the transition property? Shouldn't these values be the same?
And in that case, can we avoid code repetition?

Reply:
I applied your change request in the commit 91823b8